packages/flutter/lib/src/widgets/transitions.dart

```diff
@@ -97,6 +97,36 @@ class SlideTransition extends TransitionWithChild {
   }
 }
 
+class ScaleTransition extends StatelessComponent {
+  ScaleTransition({
+    this.child,
+    this.performance,
+    this.scale,
+    this.alignment: const FractionalOffset(0.5, 0.5)
+  });
+
+  final Widget child;
+  final PerformanceView performance;
+  final AnimatedValue<double> scale;
+  final FractionalOffset alignment;
+
+  Widget build(BuildContext context) {
+    return new BuilderTransition(
```

Comment:
Rather than using a BuilderTransition, we should just subclass TransitionWithChild, similar to the rest of the classes in this file.  It's slightly more efficient.